Xamarin.Forms.Core.UITests.Shared/Tests/ToolbarItemTests.cs

```diff
@@ -9,6 +9,8 @@ namespace Xamarin.Forms.Core.UITests
 	[TestFixture]
 	[Category(UITestCategories.ToolbarItem)]
 	[Category(UITestCategories.UwpIgnore)]
+	[Ignore("Ignoring this test as it fails sometimes on TestCloud")]
```

Comment:
the latest run of these tests
https://appcenter.ms/orgs/xtc-Xamarin-Forms/apps/AndroidControlGallery-1/test/series/pull-6437/runs

Look to now be running from PR 6437 so I think we're good to go for ToolBarItemTests once that is merged